.github/workflows/companion-deploy.yml

```diff
@@ -2,10 +2,30 @@ name: Companion Deploy
 
```

Comment:
Can we rename this to something like `Companion deploy every commit`?
https://github.com/transloadit/uppy/blob/de6bee4ea1ac2343087cba06c74d214abed053a3/.github/workflows/companion-deploy.yml#L1